### PR TITLE
Adds Hiera overrides for webserver::php

### PR DESCRIPTION
**Pull request by nickschuch**
What the title says...

### CODE REVIEW

File: manifests/php.pp [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
-  $upload_max_filesize = '20M',
-  $max_execution_time  = '300',
-  $date_timezone       = 'Australia/Sydney',
-  $error_log           = 'syslog',
+  $display_errors      = hiera('php_display_errors',      'false'),
+  $memory_limit        = hiera('php_memory_limit',        '128M'),
+  $post_max_size       = hiera('php_post_max_size',       '20M'),
+  $upload_max_filesize = hiera('php_upload_max_filesize', '20M'),
+  $max_execution_time  = hiera('php_max_execution_time',  '300'),
+  $date_timezone       = hiera('php_date_timezone',       'Australia/Sydney'),
```
Comment (cafuego) [inline manifests/php.pp]: Nope. Australia/Melbourne

File: manifests/php.pp [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
+  $error_log           = hiera('php_error_log',           'syslog'),
 
   # APC.
-  $apc_shm_size        = '64',
+  $apc_shm_size        = hiera('php_apc_shm_size',        '64'),
```
Comment (cafuego) [inline manifests/php.pp]: APC requires a unit, this should be 64M (it whines at startup)

Comment (caseyfw) [inline manifests/php.pp]: Hmmm. This got merged without being fixed.

Comment (kimpepper) [inline manifests/php.pp]: Added PR #3

**Pull request merged**